### DISCUSSION TITLE
How to select just the line not the polygon.

### DISCUSSION THREAD

**Discussion by nebsar** [pixijs/pixijs#9455] [General]
I would like to select a pixi graphics which is a polyline, line or polygon by just clicking on the line itself not the area. For example I do not want to select the polygon in the screenshot if I click inside of the polygon area. How can I do that?

![image](https://github.com/pixijs/pixijs/assets/5169314/59cdf5c5-d7e6-4384-b51f-2c2b8188be8a)

**Comment by nebsar** [comment]
I managed to select just the line by creating a buffered area alongside the lines of the polygon.

**Comment by SignDawn** [comment]
hitArea
https://www.cnblogs.com/3body/p/14981937.html

**Reply by nebsar** [reply]
Thank you for the info, but I have more complex graphics like in the screenshot, how will I generate the hit area for them?

![image](https://github.com/pixijs/pixijs/assets/5169314/7eef6329-1e01-432e-8f95-75d7c966bf48)

**Reply by nebsar** [reply]
Ok I solved the issue by adding graphics as children to the parent graphics and applying hitAreas for each of child as in the link shared. Thanks.
